web/template/layout/header.gotmpl

```diff
@@ -12,13 +12,13 @@
         <nav class="moj-header__navigation" aria-label="Account navigation">
           <ul class="moj-header__navigation-list">
             <li class="moj-header__navigation-item">
-              <a class="moj-header__navigation-link" href="{{ .SiriusURL }}/lpa" aria-current="page">LPA</a>
+              <a class="moj-header__navigation-link" href="{{ sirius "/lpa" }}" aria-current="page">LPA</a>
```

Comment:
With the normal caveat concerns about template functions, I really like this for clarity